Classes/Utility/PropertyMappingConfigurator.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the package jweiland/telephonedirectory.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace JWeiland\Telephonedirectory\Utility;
+
+use TYPO3\CMS\Extbase\Mvc\Controller\MvcPropertyMappingConfiguration;
+
+class PropertyMappingConfigurator
```

Comment:
Please add only "static" classes or classes with "static" methods to "Utility" directory. Maybe move it to Mvc/Property/Mapping/ or something else